doc/source/parameters/dem/post-processing.rst

```diff
@@ -2,7 +2,26 @@
 Post-processing
 ===============
 
-Lethe-DEM has built-in post-processing capabilities. All post-processing results are written as VTU file, generated with the same ``output frequency`` as the particles' results (see the :doc:`../dem/simulation_control` section of this guide). The name of the generated files are the same the one chosen for the particles' results plus the suffix ``-postprocess_data``. The files include:
+Lethe-DEM has built-in post-processing capabilities. All post-processing results are written as VTU, PTVU, and/or PVD files. 
```

Comment:
```suggestion
Lethe has built-in post-processing capabilities. All post-processing results are written as VTU, PTVU, and/or PVD files. 
```

Reply:
I know it was written Lethe-DEM before, but a year ago we started to replace every "Lethe-DEM" by Lethe.